javacutil/src/org/checkerframework/javacutil/AnnotationBuilder.java

```diff
@@ -153,6 +154,39 @@ public AnnotationMirror build() {
         return new CheckerFrameworkAnnotationMirror(annotationType, elementValues);
     }
 
+    /**
+     * Copies every element value from the given annotation. If an element in the given annotation
+     * doesn't exist in the annotation to be built, an error would be raised unless the element is
+     * specified in {@code ignorableElements}.
+     *
+     * @param valueHolder the annotation that holds the values to be copied
+     * @param ignorableElements the elements that can be safely dropped
+     */
+    public AnnotationBuilder copyElementValuesFromAnnotation(
```

Comment:
Return `void`? You don't say anything about the returned value, so I would have assumed you create a new AnnotationBuilder. But this is directly modifying the current object. And you don't need this as part of some fluid API, so just make it void. Or document what is returned.